src/mixins/borrow/cauldrons.js

```diff
@@ -506,7 +513,7 @@ export default {
       if (poolInterest) interest = poolInterest;
       if (!poolInterest && pool?.interest) interest = pool?.interest;
 
-      if(pool.id === 36 && this.chainId === 1) interest = pool.interest; // deprecated WBTC
+      if (pool.id === 36 && this.chainId === 1) interest = pool.interest; // deprecated WBTC
```

Comment:
Should probably leave a comment as to why we're doing this... When can we deprecate?